Web/Edubase.Services/Enums/EnumSets.cs

```diff
@@ -51,11 +52,13 @@ public static class EnumSets
             GR.GovernanceProfessionalToAFederation,
             GR.GovernanceProfessionalToAnIndividualAcademyOrFreeSchool,
             GR.GovernanceProfessionalToAMat,
+            GR.Group_SharedGovernanceProfessional,
```

Comment:
Will need to check hoiw this actually end up working

The Group_SharedGovernanceProfessional role, is one that would be applied at a MAT, and theoretically the MAT could have a pool of shared professiionals, with each establishment only using one of them - through the Establishment_SharedGovernanceProfessional role.

At the moment, I am not sure we want to add this role to the singular roles

Reply:
Verbal discussion - decision: include it now, but review it later when looking specifically at the group implementations.